js/test/date/testdatefmt_yo_BJ.js

```diff
@@ -0,0 +1,1694 @@
+            hour: 13,
+            minute: 45,
+            second: 37,
+            millisecond: 0
+        });
+        test.equal(fmt.format(date), "13:45:37");
+        test.done();
+    },
+    testDateFmtFullTimeComponentsHMSZ_yo_BJ: function(test) {
+        test.expect(2);
+        var fmt = new DateFmt({
+            locale: "yo-BJ",
+            type: "time",
+            length: "full",
+            time: "hmsz",
+            timezone: "Africa/Porto-Novo"
+        });
+        test.ok(fmt !== null);
+        
+        var date = new GregorianDate({
+            locale: "yo-BJ",
+            year: 2021,
+            month: 9,
+            day: 29,
+            hour: 13,
+            minute: 45,
+            second: 37,
+            millisecond: 0
+        });
+        test.equal(fmt.format(date), "13:45:37 WAT");
+        test.done();
+    },
+    testDateFmtFullTimeComponentsHMSAZ_yo_BJ: function(test) {
+        test.expect(2);
+        var fmt = new DateFmt({
+            locale: "yo-BJ",
+            type: "time",
+            length: "full",
+            time: "hmsaz",
+            timezone: "Africa/Porto-Novo"
+        });
+        test.ok(fmt !== null);
+        
+        var date = new GregorianDate({
+            locale: "yo-BJ",
+            year: 2021,
+            month: 9,
+            day: 29,
+            hour: 13,
+            minute: 45,
+            second: 37,
+            millisecond: 0
+        });
+        test.equal(fmt.format(date), "13:45:37 WAT");
+        test.done();
+    },
+    testDateFmtWithTimeZoneAndNoDST_yo_BJ: function(test) {
+        test.expect(2);
+        var fmt = new DateFmt({
+            locale: "yo-BJ",
+            type: "time",
+            length: "full",
+            time: "hmsz",
+            timezone: "Africa/Porto-Novo"
+        });
+        test.ok(fmt !== null);
+        
+        var date = new GregorianDate({
+            locale: "yo-BJ",
+            year: 2021,
+            month: 12,
+            day: 29,
+            hour: 13,
+            minute: 45,
+            second: 37,
+            millisecond: 0
+        });
+        test.equal(fmt.format(date), "13:45:37 WAT");
+        test.done();
+    },
+    testDateFmtFormatRelativeWithinMinuteAfter_yo_BJ: function(test) {
+        test.expect(2);
+        var fmt = new DateFmt({locale: "yo-BJ", length: "full"});
+        test.ok(fmt !== null);
+        
+        var reference = new GregorianDate({
+            locale: "yo-BJ",
+            year: 2011,
+            month: 11,
+            day: 20,
+            hour: 13,
+            minute: 45,
+            second: 0,
+            millisecond: 0
+        });
+        var date = new GregorianDate({
+            locale: "yo-BJ",
+            year: 2011,
+            month: 11,
+            day: 20,
+            hour: 13,
+            minute: 45,
+            second: 30,
+            millisecond: 0
+        });
+        test.equal(fmt.formatRelative(reference, date), "+30 s");
```

Comment:
These relative formats do not seem right. We have translations for "seconds", "minutes", etc. in `js/data/locale/yo/sysres.json` so whey aren't we using them in the relative formats?

```
    "1#in 1 second|#in {num} seconds": "#+{num} ìṣ àáy",
    "1#in 1 minute|#in {num} minutes": "#+{num} ìṣ",
    "1#in 1 hour|#in {num} hours": "#+{num} wkt",
```

Reply:
No. It's for formatrelative. not duration. so It checks the following keys which CLDR doesn't provide the translation.
```
 "1#1 second ago|#{num} seconds ago": "#-{num} s",
 "1#1 minute ago|#{num} minutes ago": "#-{num} min",
 "1#1 hour ago|#{num} hours ago": "#-{num} h",
 "1#1 day ago|#{num} days ago": "#-{num} d",
 "1#1 week ago|#{num} weeks ago": "#-{num} w",
 "1#1 month ago|#{num} months ago": "#-{num} m",
   ...
```

Reply:
What mean is that it is odd that we have the translation for the duration formats, but we don't have them for the relative formats. Maybe a special case for this?

Reply:
That's current CLDR data. Between Duration, and past/fution relative, ilib refers to different clidr file. 
For duration, ilib checks cldr file `cldr-units-full/main/*/units.json`.
But relative, ilib gets data from  cldr file `cldr-dates-full/main/*/dateFields.json`.
For example future year,  cldr-dates-full/main/yo-BJ/dateFiels.json.  it's written below:
```
"relativeTime-type-future": {
     "relativeTimePattern-count-other": "+{0} y"
},

```